src/main/java/uk/gov/moj/sdt/config/CxfConfig.java

```diff
@@ -62,22 +62,22 @@ public LoggingFeature loggingFeature() {
 
     @Bean
     public Endpoint sdtEndpoint(@Qualifier("ISdtEndpointPortType")
-                                SdtEndpointPortType sdtEndpointPortType,
-                                ServiceRequestInboundInterceptor serviceRequestInboundInterceptor,
-                                PerformanceLoggerInboundInterceptor performanceLoggerInboundInterceptor,
-                                XmlInboundInterceptor xmlInboundInterceptor,
-                                SdtUnmarshallInterceptor sdtUnmarshallInterceptor,
-                                PerformanceLoggerOutboundInterceptor performanceLoggerOutboundInterceptor,
-                                CacheSetupOutboundInterceptor cacheSetupOutboundInterceptor,
-                                XmlOutboundInterceptor xmlOutboundInterceptor,
-                                ServiceRequestOutboundInterceptor serviceRequestOutboundInterceptor,
-                                CacheEndOutboundInterceptor cacheEndOutboundInterceptor,
-                                FaultOutboundInterceptor faultOutboundInterceptor) {
+                                SdtEndpointPortType sdtEndpointPortType, ServiceRequestDao serviceRequestDao) {
+        ServiceRequestInboundInterceptor serviceRequestInboundInterceptor = new ServiceRequestInboundInterceptor(serviceRequestDao);
+        PerformanceLoggerInboundInterceptor performanceLoggerInboundInterceptor = new PerformanceLoggerInboundInterceptor();
+        XmlInboundInterceptor xmlInboundInterceptor = new XmlInboundInterceptor();
+        SdtUnmarshallInterceptor sdtUnmarshallInterceptor = new SdtUnmarshallInterceptor();
+        PerformanceLoggerOutboundInterceptor performanceLoggerOutboundInterceptor = new PerformanceLoggerOutboundInterceptor(serviceRequestDao);
```

Comment:
The ```@Component``` annotation should be removed from the corresponding interceptor classes.

The ConsumersConfig class will need to be updated with similar changes.

Reply:
yes, removed from all Interceptor classes and changed ConsumersConfig